it/controllers/PenaltiesFrontendControllerISpec.scala

```diff
@@ -424,7 +424,7 @@ class PenaltiesFrontendControllerISpec extends IntegrationSpecCommonBase with ET
     wireMockServer.findAll(postRequestedFor(urlEqualTo("/write/audit"))).asScala.toList.exists(_.getBodyAsString.contains("UserHasPenalty")) shouldBe true
   }
 
-  "NOT audit the response when the user has 0 penalties" in {
+  "audit the response when the user has 0 penalties" in {
```

Comment:
Update based on logic ie
```suggestion
  "NOT audit the response when the user has 0 LSPs and 0 LPPs" in {
```